news/2022-09-14-new-featured-artist-andy-gillion.md

```diff
@@ -0,0 +1,57 @@
+---
+layout: post
+title: "New Featured Artist: Andy Gillion"
+date: 2022-09-14 10:00:00 +0000
+---
+
+Guitarist **Andy Gillion** shines onto our Featured Artist collection!
+
+![](https://assets.ppy.sh/artists/302/header.jpg)
+
+**Andy Gillion** has been a professional guitarist for over a decade. His career launched as the lead guitarist and songwriter for metal band *Mors Principium Est*, later shifting focus towards his solo releases *Neverafter* and *Arcade Metal* — two distinct instrumental albums with one obvious commonality: *the guitar*.
+
+**13** rock/metal masterpieces are pre-timed and ready for you to map. Check out [**Andy Gillion**'s Featured Artist listing](https://osu.ppy.sh/beatmaps/artists/302) for the full list!
+
+**Andy Gillion**'s latest album was released less than 2 weeks ago, yet [-Sylvari](https://osu.ppy.sh/users/3493804) has already managed to churn out 6 difficulties for [Insert Coin](https://osu.ppy.sh/beatmapsets/1842568)! Experience **Andy Gillion**'s sound for yourself:
```

Comment:
Maybe it should be highlighted that Insert Coin is one of the songs off of the new album? It kinda read weird to me

Reply:
sumn like this
```suggestion
[-Sylvari](https://osu.ppy.sh/users/3493804) has already managed to churn out 6 difficulties for [Insert Coin](https://osu.ppy.sh/beatmapsets/1842568), one of the songs from **Andy Gillion**'s latest album that was released less than 2 weeks ago! Experience **Andy Gillion**'s sound for yourself:
```